samples/matter/common/src/event_triggers/default_event_triggers.cpp

```diff
@@ -152,7 +150,7 @@ CHIP_ERROR BlockMatterThreadCallback(TestEventTrigger::TriggerValue blockingTime
 
 		LOG_INF("Blocking Matter thread for %d seconds", blockingTimeS);
 
-		while (startTime + (blockingTimeS * 1000) > k_uptime_get()) {
+		while (startTime + static_cast<int64_t>(blockingTimeS * 1000) > k_uptime_get()) {
```

Comment:
```suggestion
		while (startTime + static_cast<int64_t>(blockingTimeS) * 1000 > k_uptime_get()) {
```
If the issue was a possible overflow then the value should be cast before the multiplication. Or it was about comparing signed and unsigned integers?

Reply:
Issue was possible overflow. Yeah, good point, I think the biggest risk of overlfow is adding the value to int64_t startTIme, so I focused on casting there. Fixed.